From e6c7de3f03c51fbdcf5ad88bf12fe9e128521f0d Mon Sep 17 00:00:00 2001
From: OPNA2608 <opna2608@protonmail.com>
Date: Fri, 19 Jul 2024 11:41:46 +0200
Subject: [PATCH] CMakeLists.txt: Fix HOST variable for ppc64

The code here defined HOST_PPC64, but the rest of the build system expects HOST_POWERPC64.

Upstream: https://github.com/libressl/portable/commit/e6c7de3f03c51fbdcf5ad88bf12fe9e128521f0d
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c496ad97b5..670aa9a1a0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -373,7 +373,7 @@ elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "mips")
 elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "powerpc")
 	set(HOST_POWERPC true)
 elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64")
-	set(HOST_PPC64 true)
+	set(HOST_POWERPC64 true)
 elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "riscv64")
 	set(HOST_RISCV64 true)
 elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "sparc64")
